myft/templates/save-for-later.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default ({ myFtApiWrite, contentId, variant, buttonText }) => {
+	return !myFtApiWrite ? null : (
```

Comment:
For readability. I think it's better to have the very large block at the end, rather in the middle; if you want to know what happens if it's false, you'd have to scroll all the way down to see that it's `null`. What do you think?